apps/dashboard/app/models/active_jobs/jobstatusdata.rb

```diff
@@ -261,6 +263,25 @@ def extended_data_sge(info)
       self
     end
 
+    # Store additional data about the job. (Fujitsu TCS specific)
+    #
+    # @return [Jobstatusdata] self
+    def extended_data_fujitsu_tcs(info)
+      return unless info.native
+      attributes = []
+      attributes.push Attribute.new "Nodes", info.native[:NODES]
+      attributes.push Attribute.new "Time Limit", pretty_time(info.wallclock_limit)
+      attributes.push Attribute.new "Submittion Time", info.native[:ACCEPT]
```

Comment:
I think this should be `Submission Time`

Reply:
Good catch, thanks!